lib/kitchen/chapter_element.rb

```diff
@@ -73,7 +73,7 @@ def abstracts
     # @param node [Nokogiri::XML::Node] the underlying node
     # @return [Boolean]
     #
-    def self.is_the_element_class_for?(node)
+    def self.is_the_element_class_for?(node, **)
```

Comment:
all of the changes like this are because this method is now passed a `config:` argument that is currently not used (will be used when we add the selector information to the `Selectors` classes).